bindings/rust/integration/benches/s2nc.rs

```diff
@@ -2,20 +2,22 @@
 // SPDX-License-Identifier: Apache-2.0
 
 use criterion::{criterion_group, criterion_main, Criterion};
-use std::{dbg, env, process::Command, time::Duration};
-mod utils;
+use std::{env, process::Command, time::Duration};
 
 pub fn s2nc(c: &mut Criterion) {
     let mut group = c.benchmark_group("s2nc");
+    /*
+    example S2NC_ARGS:
+    "--non-blocking -T -f ../integration/trust-store/ca-bundle.crt -c test_all_tls13 www.netflix.com 443"
+    Set in CriterionS2N provider in tests/integrationv2/providers.py.
+    */
```

Comment:
Now that you don't rely on the contents of S2NC_ARGS (just that it exists) the example might be unnecessary.